src/test/java/co/cask/cdap/guides/WebLogAnalyticsTest.java

```diff
@@ -88,19 +83,7 @@ public void testPageViews() throws TimeoutException, InterruptedException, IOExc
       Assert.assertEquals(4, pageViews.size());
       Assert.assertEquals(2L, (long) pageViews.get("https://accounts.example.org/signup"));
     } finally {
-      flowManager.stop();
```

Comment:
Originally I did `waitForStatus(false)` only when we're stopping the service individually instead of calling ``appManager.stopAll()` (which we do here). Though I agree that its better to explicitly wait for it to stop. Fixed.